### PR TITLE
STYLE: Remove unused import statements

### PR DESCRIPTION
**Pull request by jhlegarreta**
Remove unused import statements.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Will need to be rebased on `master` to fix potential conflicts with #2526 and #2528 once the latter get hopefully merged.

**Comment by jhlegarreta** [comment]
This PEP8 warning https://github.com/dipy/dipy/pull/2529#issuecomment-1046284682 is unrelated to the patch set; many examples do not have the imports at the top of the files. It is raised here because the file was modified here. This has already been controversial in the past (cros-ref #2103), including examples being left out of PEP8 checks, and the current inconsistency across examples. If it needs to be fixed, it should be done for all examples at once in a separate PR.

File: dipy/reconst/tests/test_eudx_dg.py [hunk 1]
```diff
@@ -1,10 +1,7 @@
-import warnings
-
 import numpy as np
 import numpy.testing as npt
 
 from dipy.direction.peaks import default_sphere, peaks_from_model
-import dipy.reconst.eudx_direction_getter
```
Comment (arokem) [inline dipy/reconst/tests/test_eudx_dg.py:7]: Weird. The name of this file and the name of the test make no sense.

**Review by arokem — COMMENTED** [review]
This is great. I think we can merge this, but along the way I noticed something strange (see comment inline). Should we post a separate issue about that?

File: dipy/reconst/tests/test_eudx_dg.py (hunk 1 above)
Comment (jhlegarreta) [reply dipy/reconst/tests/test_eudx_dg.py:7]: Thanks for the review @arokem. I do agree that the file name and contents seem not to match closely the `dipy.reconst.eudx_direction_getter.pyx`, which is the one that I guess should be targeted. Not sure about the intent of the current content: maybe the test's contents should be assimilated to the `direction.tests.test_peaks` or maybe was initially targeted to exercise `direction.closest_peak_direction_getter.pyx` (which does not have a dedicated test). To me, the pointed file should be reworked to exercise the EulerDX direction getter. Something similar happens to the `dipy.reconst.test_peak_finding.py` test (should maybe renamed to `test_recspeed.py`). Best would be to address them in a separate PR or to keep track of them through an issue.

Comment (skoudoro) [reply dipy/reconst/tests/test_eudx_dg.py:7]: Yes, let's do that in a separate PR. I will go ahead and merge this PR. Thank you @jhlegarreta

**Pull request merged**